pkg/cmd/render/render_test.go

```diff
@@ -194,10 +240,60 @@ func TestRenderCommand(t *testing.T) {
 			if err != nil {
 				t.Errorf("file %q: %v", f, err)
 			}
+			if file, ok := test.expectedContents[f]; ok {
+				data, err := ioutil.ReadFile(p)
+				if err != nil {
+					t.Errorf("error reading file %s: %v", p, err)
+					continue
+				}
+				dataJSON, err := yaml.YAMLToJSON(data)
+				if err != nil {
+					t.Errorf("error converting file %s: %v", p, err)
+					continue
+				}
+				configObj, err := runtime.Decode(unstructured.UnstructuredJSONScheme, dataJSON)
```

Comment:
`scheme.Codecs.UniversalDeserializer()`?

Reply:
the yaml dance feels weird for deserialization but I am not sure if there is a generic one for unstructured